src/main/java/org/sagebionetworks/web/client/SynapseJSNIUtilsImpl.java

```diff
@@ -697,21 +696,25 @@ private static final native boolean initFilterXss() /*-{
 						return html;
 					}
 				},
-				safeAttrValue : function(tag, name, value) {
-					// returning nothing removes the value
-					if (tag === 'img' && name === 'src') {
-						if (value
-								&& (value.startsWith('data:image/') || value
-										.startsWith('http'))) {
-							return value;
-						}
-					} else {
-						return value;
-					}
-				}
-			};
-			$wnd.xss = new $wnd.filterXSS.FilterXSS(options);
-			return true;
+                safeAttrValue: function(tag, name, value) {
+                  // Apply default safeAttrValue filtering:
+                  value = $wnd.filterXSS.safeAttrValue(tag, name, value);
```

Comment:
By default `safeAttrValue` cleans the `href` attribute. But because we overrode it, we missed that case. Use the default `safeAttrValue` first to clean href, then apply our custom rules.

I did not notice any regressions in some light testing, but I would rather aggressively filter and fix those cases if they come up.